website/content/cli/quickstart.md

```diff
@@ -167,7 +167,7 @@ options:
                             no default.
   --country COUNTRY     Country of the event. (provider: nasdaq, tradingeconomics)
   -h, --help            show this help message
-  --export EXPORT       Export raw data into csv, json, xlsx and figure into png, jpg, pdf, svg
+  --export EXPORT       Export raw data into csv, json, xlsx and figure into png, jpg, svg
```

Comment:
Might as well remove SVG as a choice, the CLI tells you a file has been created when it doesn't export anything.

Reply:
It's an issue on the Charting Extension (`figure.show(export_image=saved_path, margin=margin)`). Is this something we want to fix instead of drop?

Reply:
@hjoaquim can you debug if the export to sgv broke because of me removing svglib?

Reply:
@piiq it's not the case, just tried on dev.

Reply:
I see you already created an issue for that. Great. Let's fix it, but this is out of scope of this PR

Reply:
Agreed.